zbus_macros/tests/tests.rs

```diff
@@ -109,6 +109,7 @@ fn test_proxy() {
 
 #[test]
 fn test_derive_error() {
+    #[allow(unused)]
```

Comment:
this is a compile-only test so best to [add `ignore` annotation on it](https://doc.rust-lang.org/book/ch11-02-running-tests.html#ignoring-some-tests-unless-specifically-requested).

Reply:
ok, added